protocol/src/main/java/org/eclipse/ditto/protocol/ProtocolFactory.java

```diff
@@ -81,117 +77,54 @@ public static Adaptable setExtra(final Adaptable existingAdaptable, final JsonOb
     }
 
     /**
-     * Returns an empty {@code TopicPath}.
-     *
-     * @return the topic path.
-     */
-    public static TopicPath emptyTopicPath() {
-        return ImmutableTopicPathBuilder.empty();
-    }
-
-    /**
-     * Returns a new {@code TopicPath} for the specified {@code path}.
+     * Parses the string argument as a {@link TopicPath}.
      *
-     * @param path the path.
-     * @return the builder.
+     * @param topicPathString a String containing of the {@code TopicPath} representation to be parsed.
```

Comment:
"topicPathString a String containing {of} the {@code TopicPath} representation to be parsed"

Reply:
https://github.com/eclipse/ditto/pull/1048/commits/3d9428e1802956307b7133a32241b21301f8b9e9